specs/VersionInfo/VersionInfo.md

```diff
@@ -0,0 +1,188 @@
+
+This provides a simple but reliable and performant solution for applications, as well as class
+libraries and other components using WinAppSDK.
+
+# Examples
+
+An example using the runtime version information API to display the actual version in use at runtime:
+
+```c#
+using System;
+using System.Windows.Forms;
+using Microsoft.Windows.ApplicationModel.WindowsAppRuntime;
+
+public class MyInfo
+{
+    public static void About()
+    {
+        var build_release = Microsoft.WindowsAppSDK.Release;
+        var build_version = Microsoft.WindowsAppSDK.Runtime.Version;
+        var release = VersionInfo.ReleaseInfo.Current();
+        var runtime = .VersionInfo.RuntimeInfo.Current();
+        var message = $"Built with Windows App SDK ${build_release.Major}.${build_release.Minor}.${build_release.Patch}${build_release.FormattedVersionTag}\n"
+                      $"Minimum Windows App Runtime version ${build_version.Major}.${build_version.Minor}.${build_version.Build}.${build_version.Revision}\n"
+                      "\n" +
+                      $"Running with Windows App SDK ${release.Major}.${release.Minor}.${release.Patch}${release.FormattedVersionTag}\n" +
+                      $"Windows App Runtime version ${runtime.version.Major}.${runtime.version.Minor}.${runtime.version.Build}.${runtime.version.Revision}";
+        MessageBox.Show(message, "About MyApp", MessageBoxButtons.OK, MessageBoxIcon.Information);
+    }
+}
+```
+
+# Remarks
+
+Version information is compiled into `Microsoft.WindowsAppRuntime.Insights.Resource.dll` as the
+product is assembled (i.e. when the version is defined). The VersionInfo API retrieves this
+information via an internal API.
+
+# 6. API Details
+
+```c# (but really MIDL3)
+namespace Microsoft.Windows.ApplicationModel.WindowsAppRuntime.VersionInfo
+{
+    [contractversion(1)]
+    apicontract VersionInfoContract{};
+
+    /// Version information for the Windows App SDK release.
+    [contract(VersionInfoContract, 1)]
+    runtimeclass ReleaseInfo : Windows.Foundation.IStringable
+    {
+        ReleaseInfo();
+
+        /// Return the current release information.
+        static ReleaseInfo Current{ get; };
+
+        /// The major version of the Windows App SDK release.
+        UInt16 Major{ get; };
+
+        /// The minor version of the Windows App SDK release.
+        UInt16 Minor{ get; };
+
+        /// The patch version of the Windows App SDK release.
+        UInt16 Patch{ get; };
+
+        /// The major and minor version of the Windows App SDK release, encoded as a 32-bit integer (0xMMMMNNNN where M=major, N=minor).
+        UInt32 MajorMinor{ get; };
+
+        /// The Windows App SDK release's channel; for example, "preview", or empty string for stable.
+        String Channel{ get; };
+
+        /// The Windows App SDK release's version tag; for example, "preview2", or empty string for stable.
+        String VersionTag{ get; };
```

Comment:
>VersionTag [](http://example.com/codeflow?start=15&length=10)

Should this be an enum instead Preview, Released, LTSC, Latest, seems easer than string compare

Reply:
RECOMMEND: Remove Channel + ShortVersionTag